config/owasp/suppressions.xml

```diff
@@ -2,6 +2,21 @@
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
     <suppress>
         <notes><![CDATA[
+   file name: jackson-databind-2.9.9.3.jar
```

Comment:
I see Jackson databank older version in file ,can you remove older and see if build is green ?